tests/auxiliary/minicore.rs

```diff
@@ -41,12 +41,24 @@ macro_rules! impl_marker_trait {
 }
 
 #[lang = "pointee_sized"]
+#[diagnostic::on_unimplemented(
+    message = "values of type `{Self}` may or may not have a size",
+    label = "may or may not have a known size"
+)]
```

Comment:
Discussion: hm, this is kinda annoying because then you'd have to even keep the diagnostic attribute content in sync, but I'm not sure of a better solution so this seems reasonable.